physical/consul/consul.go

```diff
@@ -224,13 +229,47 @@ func SetupSecureTLS(ctx context.Context, consulConf *api.Config, conf map[string
 	return nil
 }
 
+// ExpandedCapabilitiesAvailable tests to see if Consul has KVGetOrEmpty and 128 entries per transaction available
+func (c *ConsulBackend) ExpandedCapabilitiesAvailable(ctx context.Context) bool {
+	available := false
+
+	maxEntries := 128
+	ops := make([]*api.TxnOp, maxEntries)
+	for i := 0; i < maxEntries; i++ {
+		ops[i] = &api.TxnOp{KV: &api.KVTxnOp{
+			Key:  c.path + "some-path-that-hopefully-does-not-exist-abc123",
```

Comment:
Can we formalize this path somehow? Maybe with a uuid const or something?

Reply:
Sure